src/pe-flyout-menu-categories/README.md

```diff
@@ -0,0 +1,41 @@
+# Flyout Menu Categories
+
+## Description
+
+> A flyout menu that dynamically displays Categories from the Service Catalog Item.
```

Comment:
The ">" can simply be removed here; it is typically used just for content blocks with no copy. Example: > None.